webpack/assets/javascripts/react_app/components/common/dates/LongDateTime.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedDate, intlShape } from 'react-intl';
+import { timezone } from '../../../common/i18n';
+
+class LongDateTime extends React.Component {
```

Comment:
I think all of these can be converted to stateless functional components.... at least my linter tells me that ;)

example: https://github.com/priley86/miq_v2v_ui_plugin/commit/7e1f1098f0f052206d21d4e448b32ad4da492b8e#diff-0588efa4575ad2864d224c7232e32c9e